app/assets/javascripts/discourse/app/components/glimmer-site-header.gjs

```diff
@@ -119,33 +123,24 @@ export default class GlimmerSiteHeader extends Component {
     }
 
     this._headerWrap = document.querySelector(".d-header-wrap");
+    this._mainOutletWrapper = document.querySelector("#main-outlet-wrapper");
     if (this._headerWrap) {
-      schedule("afterRender", () => {
-        this.headerElement = this._headerWrap.querySelector("header.d-header");
-        this.updateHeaderOffset();
-        document.documentElement.style.setProperty(
-          "--header-top",
-          `${this.headerElement.offsetTop}px`
-        );
```

Comment:
This is no longer useful. It was used in the menu panel only, but a) it always returns 0, b) was supposed to apply to mobile menu panels but wasn't used in the mobile stylesheets.